src/components/FolderTextEditor.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react'
```

Comment:
Crazy how much code a little text box can take. Good call on breaking it into its own component